moveit_core/package.xml

```diff
@@ -76,4 +76,5 @@
   <test_depend>orocos_kdl</test_depend>
   <test_depend>rosunit</test_depend>
   <test_depend>tf_conversions</test_depend>
+  <test_depend>code_coverage</test_depend>
```

Comment:
Coverage testing is not enabled for testing by default, is it?
If so, we can drop this dependency.